bytes.js

```diff
@@ -0,0 +1,47 @@
+const toHex = d => d.reduce((hex, byte) => hex + byte.toString(16).padStart(2, '0'), '')
+const fromHex = hex => {
+  if (!hex.length) return new Uint8Array(0)
+  return new Uint8Array(hex.match(/.{1,2}/g).map(b => parseInt(b, 16)))
+}
+
+const equals = (aa, bb) => {
```

Comment:
might be best to add a `aa === bb` check in the top here as a fast-path

Reply:
good call.